arctic-report-card/ARCRC.ttl

```diff
@@ -679,7 +848,8 @@ odo:ARCRC_00000061 rdf:type owl:Class ;
 ###  http://purl.dataone.org/odo/ARCRC_00000062
 odo:ARCRC_00000062 rdf:type owl:Class ;
                    rdfs:subClassOf odo:ARCRC_00000040 ;
-                   obo:IAO_0000115 "Difference in seasonal or annual air temperatures from the 1981-2010 mean (a positive anomaly is a higher temperature; a negative anomaly is a lower temperature)." ;
+                   obo:IAO_0000115 "Difference in seasonal or annual air temperatures from a reference value or long-term average (a positive anomaly is a higher temperature; a negative anomaly is a lower temperature)." ;
+                   obo:IAO_0000119 "seasonal/annual anomaly patterns for air temperature as defined in https://www.ncei.noaa.gov/access/monitoring/global-temperature-anomalies" ;
```

Comment:
Why did you change this definition? Was the previous definition wrong? Do they circumscribe the same set of members, and, if not, shouldn't we define a new class? We should discuss this one.

Reply:
I decided to change this definition because I felt the original definition was too specific since it mentioned "Difference in seasonal or annual air temperatures from the 1981-2010 mean". I changed it to "a reference value or long-term average".